tbx/project_styleguide/templates/patterns/organisms/footer/footer.html

```diff
@@ -63,6 +63,14 @@
                     {% endif %}
                 {% endwith %}
             </div>
+
+        </div>
+        <div class="grid">
+            <div class="grid__footer-address">
+                <div class="footer__address">
```

Comment:
You could consider using an `<address>` tag here - https://developer.mozilla.org/en-US/docs/Web/HTML/Element/address

Reply:
Encapsulated contact info in <address> tag in 8491e275ef2d5c86a86485ff9314c2a948014301.